conductor_api/src/static_file_server.rs

```diff
@@ -47,10 +48,22 @@ impl Future for MainFuture {
     type Error = Error;
 
     fn poll(&mut self) -> Poll<Self::Item, Self::Error> {
-        match *self {
-            MainFuture::Config(ref config) => Ok(Async::Ready(dna_connections_response(config))),
+        let response_result = match self {
+            MainFuture::Config(config) => {
+                Ok(Async::Ready(dna_connections_response(config.as_ref())))
+            }
             MainFuture::Static(ref mut future) => future.poll(),
-        }
+        };
+
+        response_result.map(|res| {
+            res.map(|mut response| {
+                response
+                    .headers_mut()
+                    .insert("Access-Control-Allow-Origin", "*".parse().unwrap());
```

Comment:
This rule is still up to debate. I don't really know what the sweet spot here is.